docs/UserGuide.md

```diff
@@ -454,8 +464,8 @@ If you don't know what the index is, refer to [Finding the index](#finding-the-i
 
 Take note that contacts cannot have tags with duplicate names. So, if you have a contact that is already tagged,
 and you decide to tag it again with the same **name** but different **value**, the new **value** will replace the old
-one! For example, if I had a contact tagged with `Friend: good`, and I tagged that same contact using another tag with
-the **name** "Friend" and the **value** "close", the pre-existing tag will be updated from `Friend: good` to `Friend: close`.
+one! For example, if a contact tagged with `Friend: good`, and is tagged using another tag with the **name** "Friend" 
```

Comment:
"if a contact **is** tagged with `Friend: good`, and is **then** tagged using another..."

Reply:
👍